src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```
@@ -0,0 +1,42 @@
+package mate.academy.springboot.datajpa.service.impl;
+
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CategoryServiceImpl implements CategoryService {
+    private final CategoryRepository categoryRepository;
+
+    @Autowired
+    public CategoryServiceImpl(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    @Override
+    public Category save(Category category) {
+        return categoryRepository.save(category);
+    }
+
+    @Override
+    public Category getById(Long id) {
+        return categoryRepository.getById(id);
```

let's throw an exception in case category isn't found